x-pack/plugins/enterprise_search/common/version.ts

```diff
@@ -8,4 +8,4 @@ import { SemVer } from 'semver';
 import pkg from '../../../../package.json';
 
 export const CURRENT_VERSION = new SemVer(pkg.version as string);
-export const CURRENT_MAJOR_VERSION = CURRENT_VERSION.major;
+export const CURRENT_MAJOR_VERSION = `${CURRENT_VERSION.major}.${CURRENT_VERSION.minor}`;
```

Comment:
@scottybollinger FYI - links using this const for URLs were previously faulty. They were generating links that looked like 

`https://www.elastic.co/guide/en/app-search/7/`
instead of
`https://www.elastic.co/guide/en/app-search/7.9/`

I should have fixed it here but let me know if you see any issues.

Also the var name is technically not correct now so IDK if we should change it lol. Is `CURRENT_MAJOR_MINOR_VERSION` too verbose? Maybe we should do `CURRENT_FULL_VERSION` and `CURRENT_VERSION`?

Reply:
Will just leave this var name for now but let me know if you want to change this and I'll open a new PR